tests/scripts/release/make_notes.py

```diff
@@ -17,19 +17,81 @@
 # under the License.
 
 import argparse
-import os
 import pickle
 from pathlib import Path
 import csv
 import sys
 from collections import defaultdict
-from typing import Callable, Dict, List, Any
 
 REPO_ROOT = Path(__file__).resolve().parent.parent.parent.parent
 sys.path.append(str(REPO_ROOT / "tests" / "scripts"))
 sys.path.append(str(REPO_ROOT / "tests" / "scripts" / "github"))
 sys.path.append(str(REPO_ROOT / "tests" / "scripts" / "jenkins"))
 
+# Tag dictionary used to create a mapping relation to categorize PRs owning same tag.
+TAG_DICT = {
```

Comment:
Thanks for coming up with this list! It'll make auto-generating the notes much simpler

Reply:
Yeah, this list will keep maintained gradualy, while monthly reports generation.